api/consumer/searchPostDocumentReference/src/v1/handler.py

```diff
@@ -34,37 +25,11 @@ def search_document_references(
     dependencies: FrozenDict[str, Any],
     logger: Logger,
 ) -> PipelineData:
-    repository: Repository = dependencies["repository"]
-
     body = fetch_body_from_event(event)
-    request_params = ConsumerRequestParams(**body)
-    assert_no_extra_params(request_params=request_params, provided_params=body)
-    nhs_number: RequestQuerySubject = request_params.nhs_number
-    pk = key(DbPrefix.Patient, nhs_number)
 
-    custodian = custodian_filter(
-        custodian_identifier=request_params.custodian_identifier
+    response = get_paginated_document_references(
+        data, ConsumerRequestParams(**body), body, dependencies, logger
     )
```

Comment:
probably best to have `request_params = ConsumerRequestParams(**body)` on it's own line so that if an error is thrown the message is more informative

Reply:
also same comment re: positional args